lib/datasource/docker/readme.md

```diff
@@ -0,0 +1,5 @@
+This datasource will fetch a release note from a source repository specified according to the [pre-defined annotation keys](https://github.com/opencontainers/image-spec/blob/main/annotations.md) of the OCI Image Format Specification.
+
+Namely, it will extract the value of label `org.opencontainers.image.source` if it exist **on the latest stable tags**.
```

Comment:
This sentence needs rewording, because right now it's unclear to me.

It will extract -> Who is doing the extracting? I think this refers to the Docker Datasource?

Reply:
```suggestion
Namely, this datasource looks for the metadata of the **latest stable** image found on the docker registry and use the value of label `org.opencontainers.image.source` as sourceUrl.
```

Reply:
```suggestion
This datasource looks for the metadata of the **latest stable** image found on the Docker registry and uses the value of the label `org.opencontainers.image.source` as the `sourceUrl`.
```

How about this?

I've capitalize the proper noun `Docker` and used monospaced font for `sourceUrl`. I've also added some `the` words to fix the grammar.